R/shiny.R

```diff
@@ -628,6 +628,15 @@ ShinySession <- R6Class(
       fun %||% identity
     },
 
+    # Ensure that `private$currentThemeDependency` is initialized. We don't do
+    # this in the `initialize()` method because then the reactiveVal doesn't
+    # have a reactive domain, and can persist after the app has exited.
+    ensureCurrentThemeDependency = function() {
+      if (is.null(private$currentThemeDependency)) {
+        private$currentThemeDependency <- reactiveVal(0)
+      }
+    },
```

Comment:
For readability purposes, I think it'd make more sense to put this definition between `getCurrentTheme` and `setCurrentTheme`